packages/wrangler/src/__tests__/publish.test.ts

```diff
@@ -7155,53 +7155,6 @@ export default{
 		});
 	});
 
-	it("should publish if the last deployed source check fails", async () => {
```

Comment:
This was removed since it tested if the `deployments` endpoint failed on publish. That API call was removed so this test is no longer necessary

Reply:
Yeah, Josh streamlined the Publish (honestly should have been another PR) it now gets the deployment ID from the response on the PUT/POST rather than making additional calls to check the ID